test/e2e/frontend/cypress/support/commands.js

```diff
@@ -69,10 +69,9 @@ Cypress.Commands.add('home', (username, password) => {
     cy.wait('@getTeam')
     cy.wait('@getTeams')
     cy.wait('@getTeamRole')
-    cy.wait('@getTeamApplications')
     cy.wait('@getInvitations')
 
-    cy.url().should('include', '/applications')
+    cy.url().should('include', '/home')
```

Comment:
```suggestion
    cy.url().should('include', '/overview')
```

Subject to accepting my proposal above